doc/conf.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+#
+# vfatqc-python documentation build configuration file, created by
+# sphinx-quickstart on Wed May 23 14:52:11 2018.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys, os
+import datetime
+import string
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.abspath('..')) # Root of the repo
+sys.path.insert(0, os.path.abspath('../macros')) # macros directory
+
+# -- General configuration -----------------------------------------------------
+
+# List all authors here
+authors = [
+    'Cameron Bravo',
+    'Mykhailo Dalchenko',
+    'Brian Dorney',
+    'Andrew Michael Levin',
+    'Louis Moureaux',
+    'Jared Sturdy',
+]
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinxcontrib.napoleon',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.doctest',
+    'sphinx.ext.pngmath',
+    'sphinx.ext.todo',
+    'sphinx.ext.viewcode']
+
+# Disable numpy docstrings for Napoleon, because they eat headers such as
+# "Examples"
+napoleon_numpy_docstring = False
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix of source filenames.
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'gempython.vfatqc-python'
```

Comment:
Should this be `vfatqc` rather than `vfatqc-python` (here and elsewhere?
The package name is `vfatqc` in the `gempyhon` namespace, although the repo name is `vfatqc-python-scripts`

Reply:
Thanks for the tip, fixed now.